crate/cli/src/tests/kms/shared/import_export_wrapping.rs

```diff
@@ -238,13 +239,13 @@ async fn test_import_export_wrap_private_key(
             wrapped_key_wrapping_data.wrapping_method,
             WrappingMethod::Encrypt
         );
-        assert_eq!(
+        assert!(
```

Comment:
Why not use `assert_eq!`: this gives better error messages when it fails AFAIK ?